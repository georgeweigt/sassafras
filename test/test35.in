* From SAS Manual, p. 730 ;

data PainRelief;
input PainLevel $ Codeine $ Acupuncture $ Relief @@;
datalines;
1 1 1 0.0	1 2 1 0.5	1 1 2 0.6	1 2 2 1.2
2 1 1 0.3	2 2 1 0.6	2 1 2 0.7	2 2 2 1.3
3 1 1 0.4	3 2 1 0.8	3 1 2 0.8	3 2 2 1.6
4 1 1 0.4	4 2 1 0.7	4 1 2 0.9	4 2 2 1.5
5 1 1 0.6	5 2 1 1.0	5 1 2 1.5	5 2 2 1.9
6 1 1 0.9	6 2 1 1.4	6 1 2 1.6	6 2 2 2.3
7 1 1 1.0	7 2 1 1.8	7 1 2 1.7	7 2 2 2.1
8 1 1 1.2	8 2 1 1.7	8 1 2 1.6	8 2 2 2.4
;

proc anova;
model Relief = PainLevel Codeine Acupuncture Codeine*Acupuncture;
