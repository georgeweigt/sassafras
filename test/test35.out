                              Analysis of Variance

    Source     DF     Sum of Squares     Mean Square     F Value     Pr > F
    Model      10        11.33500000      1.13350000       78.37     0.0000
    Error      21         0.30375000      0.01446429                       
    Total      31        11.63875000                                       

              R-Square     Coeff Var     Root MSE     RELIEF Mean
              0.973902     10.401519     0.120268        1.156250

Source                  DF       Anova SS     Mean Square     F Value     Pr > F
PAINLEVEL                7     5.59875000      0.79982143       55.30     0.0000
CODEINE                  1     2.31125000      2.31125000      159.79     0.0000
ACUPUNCTURE              1     3.38000000      3.38000000      233.68     0.0000
CODEINE*ACUPUNCTURE      1     0.04500000      0.04500000        3.11     0.0923

